plugins/modules/api.py

```diff
@@ -73,118 +136,76 @@
 
 EXAMPLES = '''
 ---
-- name: Use RouterOS API
-  hosts: localhost
-  gather_facts: no
-  vars:
-    hostname: "ros_api_hostname/ip"
-    username: "admin"
-    password: "secret_password"
-
-    path: "ip address"
-
-    nic: "ether2"
-    ip1: "1.1.1.1/32"
-    ip2: "2.2.2.2/32"
-    ip3: "3.3.3.3/32"
-
   tasks:
-    - name: Get "{{ path }} print"
+    - name: Get example - ip address print
       community.routeros.api:
         hostname: "{{ hostname }}"
         password: "{{ password }}"
         username: "{{ username }}"
-        path: "{{ path }}"
-      register: print_path
-
-    - name: Dump "{{ path }} print" output
-      ansible.builtin.debug:
-        msg: '{{ print_path }}'
+        path: "ip address"
```

Comment:
Without `register` and potential `debug` task this is not really useful. Most users don't run their playbooks with enough verbosity to see that something happened.